ui/src/app/App.test.js

```diff
@@ -223,4 +223,17 @@ describe("App", () => {
     );
     expect(wrapper.find("Login").exists()).toBe(true);
   });
+
+  it("can show the RSD link", () => {
```

Comment:
I think we should also have a test to make sure the RSD link isn't displayed

Reply:
Done.